packages/api-nest/src/repositories/entity-label.repository.ts

```
@@ -45,4 +45,30 @@ export class EntityLabelRepository implements IEntityLabelRepository {
   async save(entityLabels: EntityLabel[]): Promise<EntityLabel[]> {
     return this.repository.save(entityLabels)
   }
+
+  /**
+   * Batch find entity labels for multiple library items with label relations loaded
+   * Used by DataLoader to prevent N+1 queries
+   */
```

ESLint

[error] Missing JSDoc @returns for function.

(valid-jsdoc)